### PR TITLE
Allow empty maintainer and distributor

### PR DESCRIPTION
**Pull request by benrubson**
Hello,

This PR allows to set empty maintainer and distributor in INFO file, so that this info is not displayed in Packet Center.

Thank you 👍 

Ben

### CODE REVIEW

**Comment by jdel** [comment]
Could you read the values for 
  - maintainer
  - maintainer_url
  - distributor
  - distributor_url

from the config file ?

**Comment by benrubson** [comment]
From the config file ?
These values are taken from the `INFO` file, this PR allows for example to set in this file :
`maintainer=""`
So that this field won't be displayed in Packet Center.

However, if we don't set `maintainer` in INFO file, the default value `SSpkS` will be displayed.

**Comment by jdel** [comment]
I meant putting the default values in the configuration file. So if someone wants to change them they can just edit the config file

**Comment by benrubson** [comment]
Oh sorry, understood now :)
`JsonOutput` does not know `Config` yet, not sure how to do it properly (without breaking files under `tests/`), but I could have a look deeper.

File: lib/SSpkS/Handler/SynologyHandler.php [hunk 1]
```diff
@@ -63,7 +63,7 @@ public function handle()
         $uf = new UrlFixer($this->config->baseUrl);
         $uf->fixPackageList($filteredPkgList);
 
-        $jo = new JsonOutput();
+        $jo = new JsonOutput(this->config);
```
Comment (mbirth) [inline lib/SSpkS/Handler/SynologyHandler.php]: There's a `$` missing here.

Comment (benrubson) [inline lib/SSpkS/Handler/SynologyHandler.php]: Yep just pushed :)
Thx 👍

**Comment by benrubson** [comment]
OK, tests are failing, as I did not update them yet.

**Comment by mbirth** [comment]
Scrutinizer is still complaining. Could you fix the tests, too, please? (I think you can just give an empty Config object.)

**Comment by mbirth** [comment]
btw: You can run the checks locally by executing `vendor/bin/phpunit` from the project's root directory.

Do you know if the Synology can work with `null` values or does it need an empty string (`""`)?

**Comment by benrubson** [comment]
I tested with null values, Package Center works correctly, it simply does not show the values (expected behaviour).

**Comment by mbirth** [comment]
Looking good now. I'm still waiting for @jdel 's green light to merge all your contributions. Many thanks for them!

**Comment by benrubson** [comment]
U're welcome, thank you for your great work 👍

**Comment by benrubson** [comment]
> Do you know if the Synology can work with null values or does it need an empty string ("")?

Well, I finally faced (and reproduced) a case where Package Center was refreshing indefinitely with null values. Empty string solves this issue.

**Pull request merged**